articles/azure-monitor/app/asp-net-troubleshoot-no-data.md

```diff
@@ -280,7 +280,9 @@ For more information,
 
 ## Collect logs with dotnet-trace
 
-An alternate method of collecting logs for troubleshooting that may be particularly helpful for linux-based environments is [`dotnet-trace`](/dotnet/core/diagnostics/dotnet-trace)
+Alternatively, customers can also use a cross-platform .NET Core tool, [`dotnet-trace`](/dotnet/core/diagnostics/dotnet-trace) for collecting logs that can help further help in troubleshooting. This may be particularly helpful for linux-based environments.
+
+The command below can be executed in the [`dotnet-trace`](/dotnet/core/diagnostics/dotnet-trace) tool
```

Comment:
the command is executed in the command line, no *in the* tool.. So we need to rephrase this.